frontend/app/states/play.js

```diff
@@ -79,10 +80,32 @@ class PlayState extends Phaser.State {
       this.game.world.width / 2.5, 0,
       this.game.camera.width / 100, this.game.world.height - this.game.ground.height);
 
-    this.test = new Modal(this.game, new Keyboard(this.game));
+    //add keyboard demo button
+    this.keyboardDemoButton = new Button(this.game, null, 0,
+        this.game.world.height - 135, this.game.camera.width/12,
+        this.game.camera.width/12, "KEYBOARD", null, this.keyboardButtonCallback, this);
+
+    this.world.add(this.keyboardDemoButton);
+
+    //add label to display user input from keyboard
+    let keyboardDemoInput = new Phaser.BitmapText(this.game,
+        10, this.game.world.height - 200, "proxima_nova");
```

Comment:
same comment as above. `10` and `200` are hard-coded pixel values that we want to avoid